src/Aspire.Hosting/Dashboard/proto/resource_service.proto

```diff
@@ -191,22 +209,17 @@ message Resource {
     // The set of volumes mounted to the resource. Only applies to containers.
     repeated Volume volumes = 15;
 
-    // The aggregate health state of the resource
-    optional HealthStateKind HealthState = 16;
+    // The aggregate health state of the resource. Generally reflects data from health_reports, but may differ.
+    optional HealthStatus health_status = 16;
+    // Reports from health checks, about this resource.
+    repeated HealthReport health_reports = 19;
 
     // The resource start time.
     optional google.protobuf.Timestamp started_at = 17;
     // The resource stop time.
     optional google.protobuf.Timestamp stopped_at = 18;
```

Comment:
I believe these are all new in 9.0 so you can adjust the field numbers to not be out of order.

Reply:
Will do. FYI @snehapar9, in case you've already implemented anything in ACA for this proto.